xwiki-platform-core/xwiki-platform-tour/xwiki-platform-tour-ui/src/main/resources/TourCode/TourJS.xml

```diff
@@ -545,6 +550,9 @@ require(['jquery', 'xwiki-meta'], function ($, xm) {
 
   &amp; &gt; #tourResume {
     width: fit-content;
+    &amp;[hidden] {
+      display: none;
```

Comment:
Is this really needed?

Reply:
Without this rule, the '.btn' display would take priority over the expected [hidden] behavior. This meant that adding the attribute 'hidden' to a .btn would not change the way it was displayed, which defeats the purpose of adding this attribute.

This change restablishes the way hidden is expected to act on this .btn .